example/linux/installation.md

```
@@ -0,0 +1,240 @@
+sudo apt install jq
+```
+
+### 8. Enable and Start the Service
+
+1. Reload the systemd configuration:
+
+```bash
+sudo systemctl daemon-reload
+```
+
+2. Enable the service:
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)